src/cryptonote_core/blockchain.cpp

```diff
@@ -1773,6 +1803,15 @@ bool Blockchain::get_outs(const COMMAND_RPC_GET_OUTPUTS::request& req, COMMAND_R
   return true;
 }
 //------------------------------------------------------------------
+void Blockchain::get_output_key_mask_unlocked(const uint64_t& amount, const uint64_t& index, crypto::public_key& key, rct::key& mask, bool& unlocked)
+{
+  const auto& o_data = m_db->get_output_key(amount, index);
```

Comment:
This was taken from Blockchain::get_outs() (used by the old RPC), see line 1797.  If this is an issue, it will need sorted both places.